<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controlWidget</class>
 <widget class="QWidget" name="controlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidgetName</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,1,0,0,0" columnstretch="0,0,0,1,0">
   <item row="3" column="2" rowspan="2" colspan="3">
    <widget class="QLabel" name="label_11">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="qfe_alexcontrol.qrc">:/ALEXControl/AlternationInfo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>80</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="text">
      <string>Connect to Hardware</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>302</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBoxOptions">
     <property name="text">
      <string>&amp;More options</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLabel" name="labConfigFiles">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="text">
      <string>Save sequence</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Laser Alternation Scheme</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>continuos mode</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignCenter</set>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonA">
        <property name="text">
         <string>Acceptor</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonDA">
        <property name="text">
         <string>Donor
+
Acceptor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonD">
        <property name="text">
         <string>Donor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox_file">
     <property name="title">
      <string>SDFFPlayer sequence</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0,1,0">
        <item row="2" column="3">
         <widget class="QFEnhancedLineEdit" name="lineEdit"/>
        </item>
        <item row="2" column="4">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxTimestep">
          <property name="suffix">
           <string>Âµs</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Timestep</string>
          </property>
          <property name="buddy">
           <cstring>spinBoxTimestep</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonLoadDefault">
          <property name="text">
           <string>Load default settings</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="comboBoxSequence">
          <item>
           <property name="text">
            <string>ALEX</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ALEX slow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cw: Donor</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cw: Acceptor</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cw: Donor+Acceptor</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>external SDFF</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>current sequence</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonExternalSDFFfile">
          <property name="text">
           <string>Start &amp;External SDFF file</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>File</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Laser Power</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Voltage &amp;Donor</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxIntensityBlue</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Voltage &amp;Acceptor</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxIntensityGreen</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxIntensityBlue">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxIntensityGreen">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>350</height>
      </size>
     </property>
     <property name="title">
      <string>ALEX Sequence</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButtonPlay">
        <property name="text">
         <string>Start ALEX</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>T1</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_1</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinBox_1">
          <property name="suffix">
           <string>Âµs</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>T2</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_2</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBox_2">
          <property name="suffix">
           <string>Âµs</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>T3</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_3</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBox_3">
          <property name="suffix">
           <string>Âµs</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>T4</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_4</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="spinBox_4">
          <property name="suffix">
           <string>Âµs</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxSlow">
        <property name="text">
         <string>&amp;Slow alternation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>&lt;center&gt;&lt;b&gt;WARNING: There is no active support for this plugin!!!&lt;/b&gt;&lt;/center&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFEnhancedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfenhancedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qfe_alexcontrol.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>controlWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>controlWidget</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>604</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonConnect</sender>
   <signal>clicked()</signal>
   <receiver>controlWidget</receiver>
   <slot>connectToHardware()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxOptions</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_file</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openBla()</slot>
  <slot>openFileDialog()</slot>
  <slot>drawNew(int)</slot>
  <slot>connectToHardware()</slot>
  <slot>runSDFFOnHardware()</slot>
  <slot>runSDFF()</slot>
  <slot>runEvalWidget()</slot>
 </slots>
</ui>
